openmetadata-service/src/main/java/org/openmetadata/service/security/CatalogOpenIdAuthorizationRequestFilter.java

```diff
@@ -62,14 +63,17 @@ protected boolean isHealthEndpoint(ContainerRequestContext containerRequestConte
     return uriInfo.getPath().equalsIgnoreCase(HEALTH_END_POINT);
   }
 
-  protected String extractAuthorizedUserName(MultivaluedMap<String, String> headers) {
-    LOG.debug("Request Headers:{}", headers);
+  protected String extractAuthorizedUserName(String openIdEmail) {
+    String[] openIdEmailParts = openIdEmail.split("@");
+    return openIdEmailParts[0];
```

Comment:
should there be a exception thrown if the passedin String is not email?

Reply:
this is only for testing